game-core/src/main/java/games/strategy/triplea/ai/pro/AbstractProAi.java

```diff
@@ -86,24 +90,35 @@ public AbstractProAi(
     storedPurchaseTerritories = null;
     storedPoliticalActions = null;
     storedStrafingTerritories = new ArrayList<>();
+
+    // in case there are multiple Pro AIs running at the same time, unregister the one
+    // created before creating one so that there is only one menu item for all of the ProAI
+    DebugMenu.unregisterMenuCallback("Hard AI");
+    DebugMenu.registerMenuCallback("Hard AI", this::initialize);
   }
 
   @Override
   public void stopGame() {
     super.stopGame(); // absolutely MUST call super.stopGame() first
     calc.stop();
+
+    DebugMenu.unregisterMenuCallback("Hard AI");
```

Comment:
If the game frame is going away, presumably the menu will be destroyed and GC'd. Do we need to `unregister`? Wouldn't this happen implicitly by the `TripleAFrame` closing?